python/mxnet/ndarray/numpy/_op.py

```diff
@@ -690,10 +693,51 @@ def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None, axis
         Only returned if retstep is True
         Size of spacing between samples.
 
+
+    See Also
+    --------
+    arange : Similar to `linspace`, but uses a step size (instead of the
+             number of samples).
+    geomspace : Similar to `linspace`, but with numbers spaced evenly on a log
```

Comment:
For `see also` section, please do not include operators that we do not support yet.